dotnet/src/dotnetcore/Providers/OpenTelemetry/OpenTelemetry/GXMeterProviderBuilder.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace GeneXus.OpenTelemetry
+{
+	public static class GXMeterProviderBuilder
+	{
+		public static MeterProviderBuilder AddGxMeterAspNetInstrumentation(this MeterProviderBuilder meter)
+		{
+			meter
+				.AddAspNetCoreInstrumentation()
+				.AddHttpClientInstrumentation()
+				.AddRuntimeInstrumentation()
+				.AddOtlpExporter();
+			if (Environment.GetEnvironmentVariable("ENABLE_METRICS_CONSOLE_EXPORTER")?.ToLower() == "true")
```

Comment:
I think that we should use the same env vars as OpenTelemetry.

OTEL_LOGS_EXPORTER=console

https://opentelemetry.io/docs/specs/otel/configuration/sdk-environment-variables/

Reply:
Yes, I tried that, but i doesn´t work unless you use the Otel collector. I thought that we would loose some functionalities if we removed the ExporterConsole without any easy way to activate it in development.
On the other hand, as the documentation says, in production in general is not used.

Reply:
Or you mean to have a condition using the environment variable OTEL_LOGS_EXPORTER ?

Reply:
Exactly.
Why use a new ENv Var Name, when there is already a EnvVar for this? OTEL_LOGS_EXPORTER 

Just rename ENABLE_METRICS_CONSOLE_EXPORTER to OTEL_LOGS_EXPORTER and check if value is 'console'